Motion-CodeHouse-Ltd/backend-kwaco
# Change register payload

### What does this PR do?
This PR change register payload

### Description of Task to be completed?

- Change register payload. 
- I have replaced `rolled` with `role`

### How should be manually tested?
- Checkout branch `refactor/register-user`
- Run server, and register user to see the changes by referring to the payload below

```
{
    "full_name":"Ncuti",
    "password":"Test@123",
    "email":"ncuxdf65@gmai4.com",
    "passport_no":"00234",
    "tin":56,
    "phone":"+250723343335",
    "country":"Kenya",
    "driving_licence":"456",
    "role": "client"
}
```


### What are the relevant pivotal tracker stories?
N/A

### Screenshots
N/A
